sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/CloseHandlesInfo.java

```diff
@@ -22,6 +23,18 @@
      */
     public CloseHandlesInfo(Integer closedHandles) {
         this.closedHandles = closedHandles;
+        this.failedHandles = 0; // What should default be??
```

Comment:
Should we deprecate this?

Reply:
That is definitely another viable options to what I said in another comment.

Reply:
Pinging @JonathanGiles as this will be a common occurrence moving forward as service update and change model types, what is the strategy for handle these scenarios.

Reply:
I don't entirely understand the question. Can someone please provide a bit more context?

Reply:
@JonathanGiles This constructor is technically just there to be used by internal code but needs to be public (since it is used in the base package). The failedHandles parameter was recently added since GA and this is the new constructor we use in internal code. 

Should we deprecate the old constructor?

Reply:
One option is: We could use as a general approach for such common occurrence moving forward, is to use Fluent API approach ( using setters returns Model itself ) , ref: https://dev.to/awwsmm/build-a-fluent-interface-in-java-in-less-than-5-minutes-m7e